src/Explorer/Explorer.tsx

```diff
@@ -1754,9 +1755,12 @@ export default class Explorer {
         this.isNotebooksEnabledForAccount(true);
         return;
       }
-      const isAccountInAllowedLocation = !disallowedLocations.some(
-        (disallowedLocation) => disallowedLocation === databaseAccountLocation
-      );
+
+      // databaseAccountLocation or firstWriteLocation are not disallowed
+      const isAccountInAllowedLocation =
+        disallowedLocations.indexOf(databaseAccountLocation) === -1 ||
```

Comment:
>databaseAccountLocation [](http://example.com/codeflow?start=36&length=23)

do we need this check?

didnt we see that the control plane was using only the first write loc?

Reply:
modified it.